enumerable.rb

```diff
@@ -0,0 +1,112 @@
+module Enumerable
+  def my_each
+    i = 0
+    return enum unless block_given?
```

Comment:
- [ ] `my_each` returns an Enumerator if no block is given. Currently, its throwing the error below.
 ![image](https://user-images.githubusercontent.com/64360974/113575323-5aba1400-9626-11eb-9583-15508d06e109.png)